cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -163,7 +165,7 @@ func (command *Evaluate) Execute(args []string) (err error) {
 				language := language.Languages[languageID]
 
 				assessment, ps, err := evaluate.EvaluateRepository(command.ResultPath, model, language, command.TestdataPath, filepath.Join(languageID, repository.Name()))
-				assessmentsPerModel[model.ID()].Add(assessment)
+				assessmentsPerModelPerRepository[model.ID()][repository.Name()] = metrics.Merge(assessmentsPerModelPerRepository[model.ID()][repository.Name()], assessment)
```

Comment:
Please use and .Add

Reply:
The bigger these maps get, the worst is the copying

Reply:
And it is then easier to read what we do here

Reply:
The assessments per repository are never initialized so the object to `Add` to is `nil`. So I guess I need to manually initialize all of them first?

Reply:
We are doing that for "plain" as well, so i say, yes?